### PR TITLE
Add Event relationships

### PR DESCRIPTION
**Pull request by redmyt**
## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [x] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [x] @lhalam

## Summary of issue

add Team relation
add CustomUser(owner) relation
create migration file 0002_add_foreing_key
modify gitignore
add event urls file
add event base view

## CHECK LIST
- [x]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/event/models.py [hunk 1]
```diff
@@ -78,18 +80,39 @@ class Event(models.Model):
...
-        :return: event name, event description, event status
+        :return: event id, event team, event owner, event name,
+                 event description, event duration, event status
         """
 
-        return "{} {} {}".format(self.name, self.description, self.status)
+        return "id:{} team:{} owner:{} name:{} description:{} start_at:{}\
+                duration:{} status:{}".format(self.id,
+                                              self.team.id,
+                                              self.owner.id,
```
Comment (makarman) [inline eventually/event/models.py]: you have owner = models.ForeignKey(CustomUser, null=True)
so add self.owner.id if self.owner else None,

File: eventually/event/models.py [hunk 2]
```diff
@@ -101,34 +124,34 @@ def to_dict(self):
...
         |    'budget': 100,
         |    'status': 0,
-        |    'team': 23,
-        |    'owner': 12
         | }
         """
 
         return {
             'id': self.id,
+            'team': self.team.id,
```
Comment (makarman) [inline eventually/event/models.py:143]: you have team = models.ForeignKey(CustomUser, null=True)
so add self.team.id if self.owner else None,

File: eventually/event/models.py [hunk 3]
```diff
@@ -101,34 +124,34 @@ def to_dict(self):
...
             'name': self.name,
+            'owner': self.owner.id,
```
Comment (makarman) [inline eventually/event/models.py]: you have owner = models.ForeignKey(CustomUser, null=True)
so add self.owner.id if self.owner else None,

File: eventually/event/models.py [hunk 4]
```diff
@@ -78,18 +80,39 @@ class Event(models.Model):
...
 
-        :return: event name, event description, event status
+        :return: event id, event team, event owner, event name,
+                 event description, event duration, event status
         """
 
-        return "{} {} {}".format(self.name, self.description, self.status)
+        return "id:{} team:{} owner:{} name:{} description:{} start_at:{}\
+                duration:{} status:{}".format(self.id,
+                                              self.team.id,
```
Comment (makarman) [inline eventually/event/models.py:109]: you have team = models.ForeignKey(CustomUser, null=True)
so add self.team.id if self.owner else None,

File: eventually/event/models.py [hunk 5]
```diff
@@ -78,18 +80,39 @@ class Event(models.Model):
...
+    def __repr__(self):
+        """
+        Magic method that returns string representation of
+        event instance object.
+
+        :return: event id, event team, event owner, event name, event status
+        """
+
+        return "id:{} team:{} owner:{} name:{} status:{}".format(self.id,
+                                                                 self.team.id,
```
Comment (makarman) [inline eventually/event/models.py:92]: you have team = models.ForeignKey(CustomUser, null=True)
so add self.team.id if self.owner else None,

File: eventually/event/models.py [hunk 6]
```diff
@@ -78,18 +80,39 @@ class Event(models.Model):
...
+                                                                 self.owner.id,
```
Comment (makarman) [inline eventually/event/models.py]: you have owner = models.ForeignKey(CustomUser, null=True)
so add self.owner.id if self.owner else None,

File: .gitignore [hunk 7]
```diff
@@ -102,3 +102,6 @@ ENV/
 
 # PyCharm
 .idea/
+
+# VSCode
+.vscode/
```
Comment (lhalam) [inline .gitignore]: add endline

File: eventually/event/models.py [hunk 8]
```diff
@@ -5,10 +5,10 @@
 This module implements class that represents the event entity.
 """
 
-from django.db import models
 from django.db import IntegrityError
-from team.models import Team
+from django.db import models
```
Comment (lhalam) [inline eventually/event/models.py]: merge imports
```
from django.db import IntegrityError
from django.db import models
```

**Review by lhalam — COMMENTED** [review]
where is the migration file?

File: eventually/event/models.py (hunk 5 above)
Comment (redmyt) [reply eventually/event/models.py:92]: team id is necessary so we do not need `if` statement in this line

File: eventually/event/models.py (hunk 4 above)
Comment (redmyt) [reply eventually/event/models.py:109]: team id is necessary so we do not need `if` statement in this line

File: eventually/event/models.py (hunk 2 above)
Comment (redmyt) [reply eventually/event/models.py:143]: team id is necessary so we do not need `if` statement in this line

File: .gitignore (hunk 7 above)
Comment (redmyt) [reply .gitignore]: added

File: eventually/event/models.py (hunk 6 above)
Comment (redmyt) [reply eventually/event/models.py]: done

File: eventually/event/models.py (hunk 1 above)
Comment (redmyt) [reply eventually/event/models.py]: done

File: eventually/event/models.py (hunk 3 above)
Comment (redmyt) [reply eventually/event/models.py]: done

File: eventually/event/models.py (hunk 8 above)
Comment (redmyt) [reply eventually/event/models.py]: merged

**Comment by redmyt** [comment]
added migration file

**Review by Silent777 — APPROVED** [review]

**Pull request merged**